easybuild/easyblocks/m/mvapich2.py

```diff
@@ -70,12 +70,24 @@ def configure_step(self):
 
         # enable specific support options (if desired)
         if self.cfg['withmpe']:
-            add_configopts.append('--enable-mpe')
+            # --enable-mpe is coming from MPICH.
```

Comment:
@boegel Any comment on my idea above to have an `mpich_version = 'X.X'` in the easyconfig file?